cid/helpers/account_map.py

```diff
@@ -264,20 +263,17 @@ def create_account_mapping_sql(self, name) -> str:
         if self._metadata_source == 'dummy':
             return self.get_dummy_account_mapping_sql(name)
 
-        template_str = '''CREATE OR REPLACE VIEW  ${athena_view_name} AS
-            SELECT
-                *
-            FROM
-                ( VALUES ${rows} )
-            ignored_table_name (account_id, account_name, parent_account_id, account_status, account_email)
-        '''
-        template = Template(template_str)
+        template = Template(resource_string(
+            package_or_requirement='cid.builtin.core',
+            resource_name='data/queries/shared/account_map_org.sql',
+        ).decode('utf-8'))
+
         accounts_sql = list()
         for account in self.accounts:
             acc = account.copy()
             account_name = acc.pop('account_name').replace("'", "''")
             accounts_sql.append(
-                """ROW ('{account_id}', '{account_name}:{account_id}', '{parent_account_id}', '{account_status}', '{account_email}')""".format(account_name=account_name, **acc))
+                """ROW ('{account_id}', '{account_name}:{account_id}', '{parent_account_id}', '{account_status}', '{account_email}', '{account_email}')""".format(account_name=account_name, **acc))
```

Comment:
{account_email} is there 2 times

Reply:
Yes, second time is because it is used for account_email_id. But it might be better that account_email_id is ampty. Or if we agree to remove account_email_id column then this can be removed as well.

Reply:
second account_email is removed